scss/forms/_quantity-selector.scss

```diff
@@ -31,7 +31,14 @@
       @include button-icon($quantity-selector-icon-remove, $size: $quantity-selector-icon-width $quantity-selector-icon-remove-height, $pseudo: "after");
       order: -1;
       padding-right: $quantity-selector-btn-padding-x;
-      border-right: none;
+
+      &:not(:focus-visible) {
+        border-right: none;
+      }
```

Comment:
Sounds weird to have `:not(:focus-visible)`. Can't we have the previous behavior and add the border on the next rule ? (same below)